File: CS/CustomResourceHeaders/MainViewModel.cs
using CustomResourceHeaders.Data;
using DevExpress.Mvvm;
using System.Collections.ObjectModel;

namespace CustomResourceHeaders {
    public class MainViewModel : ViewModelBase
    {
        readonly ResourceData data;

        public MainViewModel()
        {
            data = new ResourceData();
        }
        public virtual ObservableCollection<ResourceItem> ResourceItems { get { return data.Resources; } }
    }
}

File: VB/CustomResourceHeaders/MainViewModel.vb
Imports CustomResourceHeaders.Data
Imports DevExpress.Mvvm
Imports System.Collections.ObjectModel

Namespace CustomResourceHeaders
	Public Class MainViewModel
		Inherits ViewModelBase

		Private ReadOnly data As ResourceData

		Public Sub New()
			data = New ResourceData()
		End Sub
		Public Overridable ReadOnly Property ResourceItems() As ObservableCollection(Of ResourceItem)
			Get
				Return data.Resources
			End Get
		End Property
	End Class
End Namespace
